dts/bindings/sensor/invensense,mpu6050.yaml

```diff
@@ -16,3 +16,37 @@ properties:
       The INT signal default configuration is active-high.  The
       property value should ensure the flags properly describe the
       signal that is presented to the driver.
+
+  accel-fs:
+    type: int
+    default: 2
+    description: |
+      Accelerometer full-scale range.
+      An X value represents a range of +/- X g.
+    enum:
+      - 2
+      - 4
+      - 8
+      - 16
+
+  gyro-fs:
+    type: int
+    default: 250
+    description: |
+      Gyroscope full-scale range.
+      An X value represents a range of +/- X deg/s.
+    enum:
+      - 250
+      - 500
+      - 1000
+      - 2000
```

Comment:
Are these defaults the chip reset default? If so, can you specify that in the description?

Reply:
yes, and done